BondageClub/Scripts/Validation.js

```diff
@@ -486,7 +486,7 @@ function ValidationSanitizeProperties(C, item) {
 	}
 
 	// Remove invalid properties from non-typed items
-	if (property.Type == null) {
+	if (property.Type == null && (!asset.AllowType || !asset.AllowType.length)) {
```

Comment:
I think the `property.Type == null` check is redundant here, because if `AllowType` is not set, the `Type` check on line 462 will delete it.

```suggestion
	if (!asset.AllowType || !asset.AllowType.length) {
```

Reply:
Yep, good call.